docker-compose.yml

```diff
@@ -108,15 +108,15 @@ services:
       - mock-notify
 #      - mock-os-places
       - mongodb
-      - rabbitmq
+      #- rabbitmq
     depends_on:
       - appeals-service-api-data
       - document-service-api
       - mock-horizon
       - mock-notify
 #      - mock-os-places
       - mongodb
-      - rabbitmq
+      #- rabbitmq
     volumes:
```

Comment:
why is rabbbitmq commented out in this file?

Reply:
The queue was being used to send something to backoffice. That was removed as it was causing a bug. Other than that there's no use of rabbit mq